package.json

```diff
@@ -28,9 +28,14 @@
     "nodemailer-express-handlebars": "^3.0.0",
     "pg": "^7.4.3",
     "sns-validator": "^0.3.4",
+    "update": "^0.7.4",
```

Comment:
Is this actually being used?

Reply:
Removed with no apparent consequences. I actually don't remember this specifically but I expect it's leftover clutter from minification experimentation.